src/components/selector/selector.scss

```diff
@@ -0,0 +1,8 @@
+select {
+  background-color: rgb(187, 191, 192);
+  border: 3px solid rgb(255, 255, 255);
+  border-radius: 10px;
+  height: 30px;
+  text-align-last: center;
+  width: 150px;
```

Comment:
There are still some styles missing. Look at the design. You need to blur selected option instead of changing background color of it and there is a border around dropdown box.